src/routes/BuildMyProfile/index.jsx

```diff
@@ -245,7 +246,11 @@ const BuildMyProfile = () => {
     if (basicInfo == null) {

```

Comment:
Hi @nursoltan-s,

We should not be creating traits (make the POST Api calls) if the form is empty. Notice in the screenshot I didn't select any interest yet the trait basic_Info is being created using the POST Api

![image](https://user-images.githubusercontent.com/1445965/140399486-e2776edb-033b-40f2-9762-bbe8e0e24767.png)